integrations/amplication/pyproject.toml

```
@@ -0,0 +1,113 @@
+[tool.poetry]
+name = "amplication"
+version = "0.1.0"
```

make it beta for now please
```suggestion
version = "0.1.0-beta"
```